client/my-sites/site-settings/form-general.jsx

```diff
@@ -764,6 +766,44 @@ export class SiteSettingsFormGeneral extends Component {
 		}
 	}
 
+	builtByUpsell() {
+		const { translate, site, isUnlaunchedSite: propsisUnlaunchedSite } = this.props;
+
+		// Do not show for launched sites
+		if ( ! propsisUnlaunchedSite ) {
+			return;
+		}
+
+		// Do not show if we don't know when the site was created
+		if ( ! site?.options?.created_at ) {
+			return;
+		}
+
+		// Do not show if the site is less than 4 days old
+		const siteCreatedAt = Date.parse( site?.options?.created_at );
+		const FOUR_DAYS_IN_MILLISECONDS = 4 * 24 * 60 * 60 * 1000;
+		if ( Date.now() - siteCreatedAt < FOUR_DAYS_IN_MILLISECONDS ) {
+			return;
+		}
+
+		return (
+			<Banner
+				className="site-settings__built-by-upsell"
+				title={ translate( 'We’ll build your site for you' ) }
+				description={ translate(
+					'Leave the heavy lifting to us and let our professional builders craft your compelling website.'
+				) }
+				callToAction={ translate( 'Get started' ) }
+				href="https://wordpress.com/website-building-service/?ref=unlaunched-settings"
```

Comment:
This href currently leads to a 404 🤔 Wondering if you could elaborate more on if this is expected

Reply:
Thanks for the review! This link used to work before (https://github.com/Automattic/martech/issues/1826), but I'm not sure why it leads to a 404 now. I'll check this and get back to you!